catkin_ws/src/bringup/launch/bringup.launch

```diff
@@ -3,23 +3,49 @@
     <arg name="bag" default="false" />
     <arg name="vision" default="true" />
 
+    <arg name="q_imu_auv_w" default="0.0" />
+	<arg name="q_imu_auv_x" default="0.0" />
```

Comment:
not sure if its a visual glitch but the extra indentation here is so unsatisfying lol

Reply:
looks fine in vs code